### PR TITLE
Add getTags method to Customer

### PR DESCRIPTION
**Pull request by truechernyshov**

### CODE REVIEW

**Review by subenson — DISMISSED** [review]

**Review by ashkarpetin — COMMENTED** [review]
Please update changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Security - in case of vulnerabilities.
 
 - [x] Added new properties to `Transaction`: `disputeTime`, `disputeStatus`, `isRebill`, `3ds`, `riskScore`, `retryNumber`, `isReconciled`, `hadDiscrepancy`, `hasBumpOffer`, `hasAmountAdjustment`, `billingDescriptor`
 - [x] Added new properties to `Dispute`: `transaction`, `category`, `rawResponse`, `caseId`
-- [x] Added new property to `Customer`: `revision`
+- [x] Added new property to `Customer`: `revision`, `tags`
```
Comment (ashkarpetin) [inline CHANGELOG.md:28]:
```suggestion
- [x] Added new properties to `Customer`: `revision`, `tags`
```

File: src/Entities/Customer.php [hunk 2]
```diff
@@ -375,4 +375,12 @@ public function getRevision()
...
         return $this->getAttribute('revision');
     }
+
+    /**
+     * @return array|Tag[]
+     */
+    public function getTags()
+    {
+        return $this->getAttribute('tags');
+    }
```
Comment (ashkarpetin) [inline src/Entities/Customer.php:385]:
```suggestion
    }
    
    /**
     * @param array $data
     *
     * @return array|Tag[]
     */
    public function createTags(array $data)
    {
        return array_map(function ($element) {
            return new Tag($element);
        }, $data);
    }
```

File: src/Entities/Customer.php [hunk 3]
```diff
@@ -375,4 +375,24 @@ public function getRevision()
...
+
```
Comment (ashkarpetin) [inline src/Entities/Customer.php:386]:
```suggestion

```

File: src/Entities/Customer.php [hunk 4]
```diff
@@ -375,4 +375,23 @@ public function getRevision()
...
     }
+
+    /**
+     * @return array|Tag[]
+     */
+    public function getTags()
+    {
+        return $this->getAttribute('tags');
+    }
+    /**
```
Comment (ashkarpetin) [inline src/Entities/Customer.php:386]:
```suggestion
    }

    /**
```

**Review by ashkarpetin — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by creocoder — APPROVED** [review]

**Comment by ashkarpetin** [comment]
Thanks!

**Pull request merged**